### ISSUE TITLE
cost-model is crashing with Panic error

### ISSUE THREAD

**Issue by praveenkumar0566** [kubecost/cost-analyzer-helm-chart#2341]
**Describe the bug**  
The cost model pod is crashing and restarting putting it to crashloopbackoff

```
oroutine 32660 [running]:
runtime.throw({0x347c9c7?, 0x30?})
	/usr/local/go/src/runtime/panic.go:1047 +0x5d fp=0xc01421cb70 sp=0xc01421cb40 pc=0x43919d
runtime.sigpanic()
	/usr/local/go/src/runtime/signal_unix.go:834 +0x125 fp=0xc01421cbd0 sp=0xc01421cb70 pc=0x44fb85
github.com/opencost/opencost/pkg/kubecost.isBinaryTag(...)
	/app/opencost/pkg/kubecost/kubecost_codecs.go:103
github.com/opencost/opencost/pkg/kubecost.(*AllocationSet).UnmarshalBinary(0x40dd8a?, {0x7fa5669b5000, 0xb621c, 0xb621c})
	/app/opencost/pkg/kubecost/kubecost_codecs.go:1932 +0x9d fp=0xc01421cc48 sp=0xc01421cbd0 pc=0xd1a5dd
github.com/kubecost/kubecost-cost-model/pkg/core/store.(*FileStorageStrategy[...]).Load.func1()
	/app/kubecost-cost-model/pkg/core/store/filestrategy.go:230 +0x3f fp=0xc01421cc78 sp=0xc01421cc48 pc=0x28f00df
github.com/kubecost/kubecost-cost-model/pkg/core/sys.(*RWPageFile).Read(0xc01520b830, 0xc01421cda0)
	/app/kubecost-cost-model/pkg/core/sys/rwpagefile.go:75 +0x143 fp=0xc01421cce0 sp=0xc01421cc78 pc=0x2460c63
github.com/kubecost/kubecost-cost-model/pkg/core/store.(*FileStorageStrategy[...]).Load(0x41e1840, {0xc000cb4f30, 0xc000cb4f78?})
	/app/kubecost-cost-model/pkg/core/store/filestrategy.go:230 +0x170 fp=0xc01421cde8 sp=0xc01421cce0 pc=0x28f9bf0
github.com/kubecost/kubecost-cost-model/pkg/core/store.(*FileStorageStrategy[...]).Get(0x41e1840, {0xc00b368360, 0x41ed8c0?, 0x0?}, {0x0?, 0xc00dd00ea0?, 0x0?})
	/app/kubecost-cost-model/pkg/core/store/filestrategy.go:91 +0x10c fp=0xc01421ce60 sp=0xc01421cde8 pc=0x28fa76c
github.com/kubecost/kubecost-cost-model/pkg/core/store.(*FileStorageStrategy[...]).Get({0xc020c365b0?, 0x68?, 0x0?}, {0x47a069?, 0xc0008a8a00?, 0x0?})
	/app/kubecost-cost-model/pkg/core/store/filestrategy.go:88 +0x54 fp=0xc01421ceb0 sp=0xc01421ce60 pc=0x28fe274
github.com/kubecost/kubecost-cost-model/pkg/bingen/internal/allocation.(*Store).Query.func1({0xc017f954a0?, 0xc017f954b8?})
	/app/kubecost-cost-model/pkg/bingen/internal/allocation/store.go:212 +0xba fp=0xc01421cf90 sp=0xc01421ceb0 pc=0x28e4eba
github.com/opencost/opencost/pkg/util/worker.(*queuedWorkerPool[...]).worker(0xc00fc03760)
	/app/opencost/pkg/util/worker/worker.go:117 +0x48 fp=0xc01421cfc0 sp=0xc01421cf90 pc=0x28fef28
github.com/opencost/opencost/pkg/util/worker.NewWorkerPool[...].func1()
	/app/opencost/pkg/util/worker/worker.go:72 +0x2a fp=0xc01421cfe0 sp=0xc01421cfc0 pc=0x28f16ca
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1598 +0x1 fp=0xc01421cfe8 sp=0xc01421cfe0 pc=0x46f641
created by github.com/opencost/opencost/pkg/util/worker.NewWorkerPool[...]
	/app/opencost/pkg/util/worker/worker.go:72 +0x153

```
**Expected behavior**  
The pod shouldn't be crashing

**Screenshots**  

**What impact will this have on your ability to get value out of Kubecost?**
UI isn't coming up due to pod crash

**Bug report**
Bug report in the ticket
https://kubecost.zendesk.com/agent/tickets/4317

**Comment by thomasvn** [comment]
@praveenkumar0566 Thanks for reporting! We've logged this as a bug and plan on fixing in the next release. We'll reach back out if more context needed on the panic.

**Comment by praveenkumar0566** [comment]
@thomasvn For your awareness, Another occurrence of panic on 1.104.1
https://kubecost.zendesk.com/agent/tickets/4456

**Comment by nikovacevic** [comment]
This looks to me like there's a file on disk that's gotten corrupted, so bingen in choking on unmarshaling it. (It's seemingly not a general issue, or else all bingen unmarshaling would be borked.)

@praveenkumar0566 any chance we can get an ETL dump for this?

**Comment by praveenkumar0566** [comment]
Hi @nikovacevic sent you over DM in slack

**Comment by chipzoller** [comment]
Internal ticket says fixed as of 1.105.
